app/routes/endemics/which-type-of-review.js

```diff
@@ -66,8 +66,15 @@ module.exports = [
       },
       handler: async (request, h) => {
         const { typeOfReview } = request.payload
+        const { latestVetVisitApplication, previousClaims } = getEndemicsClaim(request)
+
         setEndemicsClaim(request, typeOfReviewKey, claimType[typeOfReview])
 
+        // If user has an old world application within last 10 months and a review claim and this is his first follow-up claim
+        if (latestVetVisitApplication && previousClaims && previousClaims?.length === 1) {
```

Comment:
```suggestion
        if (latestVetVisitApplication) {
```
The latestVetVisitApplication is the 1 review claim - you don't need to check for additional review claims. Is there logic somewhere to hide this screen once the user has entered the data?